### PR TITLE
Allows crew to spawn with more mechanical or assisted organs.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
As per title.

### CODE REVIEW

**Comment by KasparoVy** [comment]
HELL YEAH

File: code/modules/client/preference_setup/general/03_body.dm [hunk 1]
```diff
@@ -617,15 +617,21 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 		return TOPIC_REFRESH
 
 	else if(href_list["organs"])
-		var/organ_name = input(user, "Which internal function do you want to change?") as null|anything in list(BP_HEART, BP_EYES)
+		var/organ_name = input(user, "Which internal function do you want to change?") as null|anything in list("Heart", "Eyes", "Lungs", "Liver", "Kidneys")
```
Comment (KasparoVy) [inline code/modules/client/preference_setup/general/03_body.dm]: Any reason we ain't just using the defines for all of 'em? Not using defines seems like a bit of a regression

Comment (NonQueueingMatt) [reply code/modules/client/preference_setup/general/03_body.dm]: Because these are display names and we want them capitalized

Comment (KasparoVy) [reply code/modules/client/preference_setup/general/03_body.dm]: Thanks for clearing that up, makes sense

**Comment by NonQueueingMatt** [comment]
!review

File: code/modules/organs/organ.dm [hunk 2]
```diff
@@ -305,23 +305,17 @@
 
 /obj/item/organ/proc/robotize() //Being used to make robutt hearts, etc
 	robotic = 2
-	src.status &= ~ORGAN_BROKEN
```
Comment (Alberyk) [inline code/modules/organs/organ.dm]: Why is this stuff being removed?

Comment (NonQueueingMatt) [reply code/modules/organs/organ.dm]: Setting the flag to = ORGAN_ROBOT should already do everything that does, no?

**Pull request merged**